content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 50
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you will install the [Kiali](https://www.kiali.io) add-on and use
+the web-based graphical user interface for viewing service graphs of the mesh
+as well as your Istio configuration objects such as virtual services
+and destination rules. You will also use the Kiali API to generate graph data
+in the form of consumable JSON.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used as
+the example application throughout this task.
+
+## Before you begin
+
+{{< idea_icon >}} If you already have Istio installed and need Kiali installed, you can follow the [Kiali install instructions here](https://www.kiali.io/gettingstarted/).
+
+Before you install [Istio](/docs/setup/) in your cluster, you first need to create a secret in your Istio namespace with credentials that users will use to authenticate to Kiali. See the [Helm README](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/README.md#installing-the-chart) for details, but here is a command you can run to create a secret:
+
+{{< text ascii >}}
+USERNAME=$(echo -n 'admin' | base64) ; \
+  PASSPHRASE=$(echo -n 'mysecret' | base64) ; \
+  NAMESPACE=istio-system ; \
+  kubectl create namespace $NAMESPACE ; \
+  cat <<EOF | kubectl apply -f -
```

Comment:
26-30 start the line with $ and remove the trailing `; \`

IOW make them separate commands.

Reply:
This is done. I also changed it - no longer uses text bash (since it wasn't rendering the "here document" properly), I changed it to use "```bash" and that looks better and the reader can still click the "copy" button and get all the commands on the clipboard so it can be pasted to a console.

![image](https://user-images.githubusercontent.com/2029470/46369691-ae140b80-c651-11e8-84a2-a8d2351090a0.png)